torch/special/__init__.py

```diff
@@ -15,15 +16,81 @@
     \text{out}_{i} = \ln \Gamma(|\text{input}_{i}|)
 """ + """
 Args:
-    input (Tensor): the input tensor.
+    {input}
 
 Keyword args:
-    out (Tensor, optional): the output tensor.
+    {out}
 
 Example::
 
     >>> a = torch.arange(0.5, 2, 0.5)
     >>> torch.special.gammaln(a)
     tensor([ 0.5724,  0.0000, -0.1208])
 
-""")
+""".format(**common_args))
+
+erf = _add_docstr(_special.special_erf,
+                  r"""
+erf(input, *, out=None) -> Tensor
+
+Computes the error function of each element. The error function is defined as follows:
```

Comment:
"of each element of :attr:\`input\`" would be more precise.

Alternatively this could say "Computes the error function of :attr:\`input\`." to be more direct.